core/components/organisms/navigation/HorizontalNavigation.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Text, Icon, Pills } from '@/index';
+import { NavigationProps } from '@/index.type';
+import {
+  getTextAppearance,
+  getIconAppearance,
+  getPillsAppearance,
+  isActive,
+  Menu,
+} from './utils';
+
+export interface HorizontalNavigationProps {
+  menus: NavigationProps['menus'];
+  active?: NavigationProps['active'];
+  onClick?: NavigationProps['onClick'];
+}
```

Comment:
If you are not making any modifications, I would suggest you to pick props directly from NavigationProps using `Pick`.